MekHQ/src/mekhq/campaign/autoResolve/helper/UnitStrengthHelper.java

```diff
@@ -0,0 +1,23 @@
+package mekhq.campaign.autoResolve.helper;
+
+import megamek.common.*;
+import mekhq.campaign.autoResolve.scenarioResolver.components.UnitStrength;
+
+public class UnitStrengthHelper {
+
+    static public UnitStrength getUnitStrength(Entity entity) {
+        // The values here are completely arbitrary and should be adjusted as needed, probably should be in a config file.
+        if (entity instanceof Mek) {
+            return new UnitStrength(entity.calculateBattleValue(), +1, entity);
```

Comment:
It might be worth taking a look at `entity.getGenericBattleValue()` - GBV is static per unit type & weight, so you might be able to do some math to derive inter-unit type power modifier from that. I don't think the arbitrary values are inherently wrong, mind, just wanted to give you options. :)